Dockerfile.fuzz

```diff
@@ -0,0 +1,16 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0-alpine as builder
```

Comment:
why could not we just use Restler Fuzzer docker image from Microsoft https://hub.docker.com/_/microsoft-restlerfuzzer-restler instead?

Reply:
it looks like microsoft does not update the restlerfuzzer image as often as the github repo, I can update that later when they have updated one. Currently the dockerfile is based on the latest dockerfile from restler repo.